test/zdtm/static/Makefile

```diff
@@ -222,6 +222,7 @@ TST_NOFILE	:=				\
 		child_subreaper_and_reparent	\
 		memfd00				\
 		memfd01				\
+		memfd02				\
```

Comment:
but the test is in another commit, isn't it?

Reply:
Good catch!
Fixed

Thank you for being thorough